src/status_im/chat/models/link_preview.cljs

```diff
@@ -27,6 +27,15 @@
                #{})
              {})))
 
+(fx/defn resolve-community-info
+  {:events [::resolve-community-info]}
+  [cofx community-id]
+  (fx/merge cofx
+            {::json-rpc/call [{:method     (json-rpc/call-ext-method "requestCommunityInfoFromMailserver")
+                               :params     [community-id]
+                               :on-success #()
```

Comment:
no need to pass a function if you don't want to do anything, or perhaps just `log/debug` something?

Reply:
Well, unfortunately if you omit it, you will get a crash. And I had it :)

Reply:
oh I see :) fair enough